package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "usabilla-api",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
As a good practice better change the version at a different commit. Just before publishing. Please revert this change!